src/Core/Collections/Leaderboard/Leaderboard.tsx

```diff
@@ -19,32 +24,36 @@ type CachedRankings = {
   };
 };
 
-const LeaderboardTimeRangesMap = {
-  [LeaderboardTimeRange.ALL_TIME]: {
-    label: 'All time',
-    tooltip: 'Points collected for all time',
-  },
-  [LeaderboardTimeRange.WEEK]: {
-    label: 'Weekly',
-    tooltip: `Points collected for the current week - ${moment().startOf('week').format('MMMM D')} to ${moment()
-      .endOf('week')
-      .format('MMMM D')}`,
-  },
-  [LeaderboardTimeRange.MONTH]: {
-    label: 'Monthly',
-    tooltip: `Points collected for the current month - ${moment().startOf('month').format('MMMM D')} to ${moment()
-      .endOf('month')
-      .format('MMMM D')}`,
-  },
-  [LeaderboardTimeRange.IGBO_VOICE_ATHON]: {
-    label: 'Igbo Voice-athon',
-    tooltip: 'Points collected for the Igbo Voice-athon - July 24 to October 24',
-  },
-};
+// const LeaderboardTimeRangesMap = {
+//   [LeaderboardTimeRange.ALL_TIME]: {
+//     label: 'All time',
+//     tooltip: 'Points collected for all time',
+//   },
+//   [LeaderboardTimeRange.WEEK]: {
+//     label: 'Weekly',
+//     tooltip: `Points collected for the current week - ${moment().startOf('week').format('MMMM D')} to ${moment()
+//       .endOf('week')
+//       .format('MMMM D')}`,
+//   },
+//   [LeaderboardTimeRange.MONTH]: {
+//     label: 'Monthly',
+//     tooltip: `Points collected for the current month - ${moment().startOf('month').format('MMMM D')} to ${moment()
+//       .endOf('month')
+//       .format('MMMM D')}`,
+//   },
+//   [LeaderboardTimeRange.IGBO_VOICE_ATHON]: {
+//     label: 'Igbo Voice-athon',
+//     tooltip: 'Points collected for the Igbo Voice-athon - July 24 to October 24',
+//   },
+// };
 
 const Leaderboard = (): ReactElement => {
-  const [leaderboard, setLeaderboard] = useState<LeaderboardType>(LeaderboardType.RECORD_EXAMPLE_AUDIO);
-  const [leaderboardTimeRange, setLeaderboardTimeRange] = useState<LeaderboardTimeRange>(LeaderboardTimeRange.ALL_TIME);
+  // const [leaderboard, setLeaderboard] = useState<LeaderboardType>(LeaderboardType.RECORD_EXAMPLE_AUDIO);
+  const [leaderboardIndex, setLeaderboardIndex] = useState<number | null>();
```

Comment:
for clarity, instead of this being a number, it should be a `LeaderboardType`

```suggestion
  const [selectedLeaderboard, setSelectedLeaderboard] = useState<LeaderboardType | null>(null);
```

Reply:
I wouldn't want to do an `Array.find` each time i need to access the selected LeaderboardItem. Using an index runs `O(1)`. For the sake of clarity I will update this to be the actual selected leaderboard object.